app/main.py

```diff
@@ -1,8 +1,45 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(
+            self, comfort_class: int,
+            clean_mark: int,
+            brand: str
+    ) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(
+           self, distance_from_city_center: float,
+           clean_power: int,
+           average_rating: float,
+           count_of_ratings: int
+    ) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: list) -> float:
```

Comment:
What if input data consists of cars with `clean_mark > clean_power`.
Your `serve_cars()` method returns 0 in such case, which is not float

Reply:
I don`t understand what mean this comment. I need Any type to return?

Reply:
or i need total income 0.0?

Reply:
> I don`t understand what mean this comment. I need Any type to return?

Yes !

Reply:
> > I don`t understand what mean this comment. I need Any type to return?
> 
> No , sorry , just think what type is `0` , because you can return it too )
0 type we can use as integer , bool etc. But in this moment the total_income variable is initialized as a float (total_income = 0), and the function returns the value of total_income after the loop (return round(total_income, 1)). the return value will be 0.0 since total_income is set to 0 and is not modified in the loop. And for this I think that type of 0 is float too. this my mind for that , is it true? or I don`t understand something?

Reply:
> > I don`t understand what mean this comment. I need Any type to return?
> 
> No , sorry , just think what type is `0` , because you can return it too )

0 type we can use as integer , bool etc. But in this moment the total_income variable is initialized as a float (total_income = 0), and the function returns the value of total_income after the loop (return round(total_income, 1)). the return value will be 0.0 since total_income is set to 0 and is not modified in the loop. And for this I think that type of 0 is float too. this my mind for that , is it true? or I don`t understand something?

Reply:
I'm talking about a case when your check doesn't work (And this is potentially possible)

So right return type here will be :

`float | int:`

Reply:
oh, ok understand that , thanks.  now i add this type

Reply:
for moment when i write , idk that we can use two types

Reply:
Oh yeah , it can be 2 and more !)